.github/workflows/ci.yml

```diff
@@ -49,17 +49,18 @@ jobs:
       fail-fast: true
       matrix:
         os:
-          - { label: ubuntu-20.04, code: focal }
+          - { label: ubuntu-latest, code: latest }
         compiler:
           - { compiler: GNU10,  CC: gcc-10,   CXX: g++-10,     packages: gcc-10 g++-10 }
-          - { compiler: LLVM12, CC: clang-12, CXX: clang++-12, packages: clang-12 libomp-12-dev libclang-common-12-dev llvm-12-dev clang++-12 libc++-12-dev libc++1-12 libc++abi1-12 lld-12}
+          - { compiler: LLVM14, CC: clang-14, CXX: clang++-14, packages: clang-14 libomp-14-dev libclang-common-14-dev llvm-14-dev clang++-14 libc++-14-dev libc++1-14 libc++abi1-14 lld-14}
```

Comment:
This should be LLVM15, or LLVM16 even.

Reply:
OK. I would suggest accepting this PR now, and if there is a consensus to use the latest LLVM (not available from the system repositories), I will prepare a new PR installing it from https://apt.llvm.org/

@TurboGit What's your view on this?

Reply:
I would really only concentrate on latest stable releases. So to me LLVM14 seems like a good candidate at this point as the latest available on Debian/experimental is LLVM15.